File: apiEZPZ/src/main/java/com/example/apiezpz/prohibited/domain/Prohibit.java
package com.example.apiezpz.prohibited.domain;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
@Table(name = "prohibit")
public class Prohibit {

    @Column(name = "GUBUN")
    private String gubun;
    @Column(name = "CARRY_BAN")
    private String carryBan;
    @Column(name = "CABIN")
    private String cabin;
    @Column(name = "TRUST")
    private String trust;
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "SEQ")
    private Long seq;


}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/prohibited/domain/Prohibit.kt
package com.example.apiezpz.prohibited.domain

import jakarta.persistence.*

@Entity
@Table(name = "prohibit")
data class Prohibit(
    @Column(name = "GUBUN")
    var gubun: String? = null,
    
    @Column(name = "CARRY_BAN")
    var carryBan: String? = null,
    
    @Column(name = "CABIN")
    var cabin: String? = null,
    
    @Column(name = "TRUST")
    var trust: String? = null,
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "SEQ")
    var seq: Long? = null
) 